app/router.js

```diff
@@ -14,14 +14,15 @@
  * limitations under the License.
  */
 
-import Ember from 'ember';
-import config from './config/environment';
+const Ember = require('ember');
+const config = require('./config/environment');
 
-var Router = Ember.Router.extend({
+const Router = Ember.Router.extend({
   location: config.locationType
 });
 
-Router.map(function() {
+// eslint-disable-next-line array-callback-return
```

Comment:
How does this rule relate to the next line?

Reply:
The other alternative is to `return null` at the en d. Do you have a better idea for a fix here?

Reply:
I was just confused because `Router` isn't an array, but I see that under "known limitations" for this rule, eslint even mentions that it applies to _any_ method with a similar name to array methods... the disabling comment is probably the best way 👍

Reply:
👍